### ISSUE TITLE
7.4.x pluggy CI builds fail

### ISSUE THREAD

**Issue by The-Compiler** [pytest-dev/pytest#11293]
see e.g. https://github.com/pytest-dev/pytest/actions/runs/5772741776/job/15648272495?pr=11286

```
INFO: pip is looking at multiple versions of pluggy to determine which version is compatible with other requirements. This could take a while.
py37-pluggymain-pylib-xdist: exit 1 (11.93 seconds) /home/runner/work/pytest/pytest> python -I -m pip install 'pluggy@ git+https://github.com/pytest-dev/pluggy.git' 'py>=1.8.2' 'pytest-xdist>=2.1.0' -e . pid=1883
  Running command git clone --filter=blob:none --quiet https://github.com/pytest-dev/pluggy.git /tmp/pip-install-i9cw8ots/pluggy_51a0b7decb294cd0865e1b815f453f8b
ERROR: Package 'pluggy' requires a different Python: 3.7.17 not in '>=3.8'
  py37-pluggymain-pylib-xdist: FAIL code 1 (12.27 seconds)
```

Not sure what to do there... should we just upgrade those pluggy builds from py37 to py38 for CI?

**Comment by bluetech** [comment]
> should we just upgrade those pluggy builds from py37 to py38 for CI?

IMO, yes, should just change py37 -> py38, since the idea is to test against the latest pluggy.

**Comment by Pierre-Sassoulas** [comment]
Closing as fixed in #11294
